src/views/FfaListingView.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <section id='single-listing'>
+    <div v-if="!isReady">
+      <div class='tile is-vertical'>
+        <div class='tile message'>
+          Connecting to blockchain
+        </div>
+        <div class='tile'>
+        <font-awesome-icon
+          class="fa-spin"
+          :icon="['fab', 'ethereum']"
+          :class="svgColorClass"/>
+        </div>
+      </div>
+    </div>
+    <div v-if="isReady">
```

Comment:
Can use `v-else` here. Also can switch conditionals to dispense of the `!` in the if statement. Although this isn't a big deal here